### PR TITLE
Jeremy/eng 1066

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
add additional info to Admin Booking page

## How to Test
- [ ] Switch backend to https://github.com/thebeetoken/beenest-backend/pull/793
- [ ] Visit admin/bookings/:id
- [ ] See the following info added:

```
Guest Full Name

Guest Email

Guest phone number

Dates Booking

Number of Guests

Price/ Night

Total Amount

```

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/components/Admin/AdminBookings/AdminBookingsTable/AdminBookingStatus/AdminBookingStatus.tsx [hunk 1]
```diff
@@ -37,6 +37,7 @@ const StatusHandler = ({ type, person }: StatusHandlerProps) => {
 
 const renderStatus = (props: Booking) => {
   const { status, rejectedBy, cancelledBy } = props;
+  console.log('props:', props);
```
Comment (tc) [inline src/components/Admin/AdminBookings/AdminBookingsTable/AdminBookingStatus/AdminBookingStatus.tsx]: extra console log

File: src/components/Admin/AdminBookings/AdminBooking/AdminBooking.tsx [hunk 2]
```diff
@@ -31,17 +32,23 @@ const AdminBooking = ({ match }: any): JSX.Element => (
...
                 </p>
               </div>
               <div>
                 <p>
-                  Listing: <BeeLink to={`/admin/listings/${booking.listingId}/edit`}>{booking.listingId}</BeeLink>
+                  Listing: <BeeLink to={`/admin/listings/${booking.listing.id}/edit`}>{booking.listing.id}</BeeLink>
                 </p>
               </div>
               <div>
+                <p>Status: {getAdminBookingDisplayStatus(booking.status)}</p>
```
Comment (tc) [inline src/components/Admin/AdminBookings/AdminBooking/AdminBooking.tsx]: did amy want to see the guest info too? email, etc

Comment (jerminatorhits) [reply src/components/Admin/AdminBookings/AdminBooking/AdminBooking.tsx]: ya will add 🙏

File: src/components/Admin/AdminBookings/AdminBookingsTable/AdminBookingStatus/AdminBookingStatus.tsx (hunk 1 above)
Comment (jerminatorhits) [reply src/components/Admin/AdminBookings/AdminBookingsTable/AdminBookingStatus/AdminBookingStatus.tsx]: ty removed

**Pull request merged**